src/indent.c

```diff
@@ -123,14 +123,22 @@ tabstop_padding(colnr_T col, int ts_arg, int *vts)
 
 /*
  * Find the size of the tab that covers a particular column.
+ *
+ * If this is being called as part of a shift operation, col is not the cursor
+ * column but is the column number to the left of the first non-whitespace
+ * character in the line.  If the shift is to the left (left = 1), then return
```

Comment:
Should `left` use `TRUE` and `FALSE` instead of `0` and `1`? `TRUE` and `FALSE` are passed to the `left` argument of `shift_line()`, for example in indent.c:
```c
		shift_line(TRUE, FALSE, 1, TRUE);
```

Reply:
I think you're right. Fixed.